.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: objective-c
+
+notifications:
+  email:
+    on_success: never
+    on_failure: change
+
+script: 'curl -s https://raw.githubusercontent.com/atom/ci/master/build-package.sh | sh'

```

Comment:
all examples I found are either doing this way or duplicating the build-package;sh in their repository. As we don't have to modify it for now, starting with just a reference seems fine